QuizAppNew/View/QuestionView.swift

```diff
@@ -0,0 +1,182 @@
+                Text(question.optionC)
+                    .foregroundColor(Color.black)
+                    .padding([.horizontal,.vertical],5)
+                    .frame(maxWidth: .infinity)
+                    .background(
+                        RoundedRectangle(cornerRadius: 15)
+                            .stroke(color(option: question.optionC), lineWidth: 1)
+                    )
+            })
+            
+            //FOR OPTIONS....D
+            
+         //   Button(action: {selected = question.optionD}, label: {
+            Button(action: {}, label: {
+                Text(question.optionD)
+                    .foregroundColor(Color.black)
+                    .padding([.horizontal,.vertical],5)
+                    .frame(maxWidth: .infinity)
+                    .background(
+                        RoundedRectangle(cornerRadius: 15)
+                            .stroke(color(option: question.optionD), lineWidth: 1)
+                    )
+            })
+            
+            Spacer(minLength: 0)
+            
+            //Buttons use for submitted answer And Next Question use with HStack...
+            
+            HStack(spacing: 15) {
+                
+            //    Buttons use for submitted answer
+                
+                Button(action: checkAns, label: {
+                    Text("Submit")
+                        .fontWeight(.heavy)
+                        .foregroundColor(.white)
+                        .padding(.vertical)
+                        .frame(maxWidth: .infinity)
+                        .background(Color.blue)
+                        .cornerRadius(15)
+                })
+                
+                // disabling
+                .disabled(question.isSubmitted ? true : false)
+                .opacity(question.isSubmitted ?  0.7 : 1)
+                
+                // For next question Button....
+                
+                Button(action: {
+                    withAnimation{
+                        question.completed.toggle()
+                        answered += 1
+                    }
+                }, label: {
+                    Text("Next")
+                        .fontWeight(.heavy)
+                        .foregroundColor(.white)
+                        .padding(.vertical)
+                        .frame(maxWidth: .infinity)
+                        .background(Color.blue)
+                        .cornerRadius(15)
+                })
+                .disabled(!question.isSubmitted ? true : false)
+                .opacity(!question.isSubmitted ? 0.7 : 1)
+            }
+            .padding(.bottom)
+            
+        }
+        .padding()
+        .background(Color.white)
+        .cornerRadius(25)
+        .shadow(color: Color.black.opacity(0.05), radius: 5, x: 5, y: 5)
+        .shadow(color: Color.black.opacity(0.05), radius: 5, x: -5, y: -5)
+    }
+    
+    //MARK: highlighting answer use color
```

Comment:
are you fucking idiot why use business logic?

Reply:
removed